anomalib/data/utils/split.py

```diff
@@ -116,3 +125,14 @@ def random_split(
     # outer list: subsets with the given ratio, inner list: per-label unique
     subsets = list(map(list, zip(*subsets)))
     return [concatenate_datasets(subset) for subset in subsets]
+
+
+def split_normal_and_anomalous(dataset: AnomalibDataset) -> Tuple[AnomalibDataset, AnomalibDataset]:
```

Comment:
a bit verbose, but how about renaming this to `split_to_normal_and_anomalous`. By reading the name, I was not clear what exactly it does.

Reply:
How about `split_by_label`?